lib/project_extensions.dart

```diff
@@ -0,0 +1,5 @@
+class ProjectExtensions {
+  const ProjectExtensions({this.steps = const [4, 8, 12, 16, 20, 24, 32, 48, 64]});
+
+  final List<int> steps;
```

Comment:
what are the steps? add doc comments on public api in package to help users understand the stuff better (I am seeing this first time, so please understand confusion and sorry for silly questions - but that's how folks will go through it)

Reply:
I added some description and also while doing this i figured out that will be better to use Set instead of List; to be sure that numbers are unique ( it reduces potential mistake if dev type the same step value )